frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -1030,7 +1030,10 @@ function ReaderPaging:onGotoPageRel(diff)
             goto_end(x)
         elseif new_page > 0 then
             self:_gotoPage(new_page)
+            new_va = self.visible_area:copy()
+            old_va = self.visible_area
             goto_end(y, -y_diff)
+            goto_end(x, -x_diff)
         else
             goto_end(x)
         end
```

Comment:
(I'm still not familiar enough with readerpaging & co to be sure this is the right fix) but it doesn't _read_ too bad :)
The `goto_end(x, -x_diff)` looks logical.
The `new_va =` and `old_va =` feels like they could make sense to cleanup positionning and start afresh if we go to a new page. May be just add a small comment above these 2 lines, something like:
`-- be sure the new and old view areas are reset so that... [what would be happening if we would not reset]`